proposals/collection-literals.md

```diff
@@ -478,7 +468,7 @@ Not having a *known length* does not prevent any result from being created. Howe
             T __result = new T(capacity: __len);
 
             __result.Add(__e1);
-            __result[__k1] = __v1;
+            __result.Add(new T1(__k1, __v1));
```

Comment:
this is interesting.  it would allow a list of Foo's where Foo might have a constructor that takes two expressions.  Or is there a rule later that says this only applies if T1 is KVP?

Reply:
Answered below.